### ISSUE TITLE
Stated swift package manager support is an overstatement

### ISSUE THREAD

**Issue by doozMen** [joreilly/PeopleInSpace#146]
I made a small experiment using the `./gradlew createSwiftPackage` to generate a swift package of `PeopleInSpace`. This swift package I then tried to use as a local swift package. Only this does not work.

As I reference I have this project where I added a very basic setup https://github.com/doozMen/KMM-PeopleInSpace

If you agree may I suggest to not suggest that this project uses swift package manager as it is a good attempt but does not actually compile?

**Comment by joreilly** [comment]
Are you trying this on an M1 device....if so you might be running in to https://github.com/ge-org/multiplatform-swiftpackage/issues/31.  I believe there are some forks of this that support M1 but I haven't had chance to investigate yet.  The related changes I made at the time were done as part of exploration (described in https://johnoreilly.dev/posts/kotlinmultiplatform-swift-package/) at the time to see what was possible.....default for project though is still not using that approach.

**Comment by doozMen** [comment]
@joreilly thanks for mentioning but I think it is more then the M1 that lies to the problem. The xcframework produced also only has simulator architecture which would make impossible to build on any iOS device with it.

**Comment by joreilly** [comment]
Hmm, could be issue with config I have currently there....though the library (https://github.com/ge-org/multiplatform-swiftpackage) states that it should support device and simulator.  It's been some time since I tried this out and I could definitely have missed something.  As mentioned, this is not default approach for the sample and was mostly added, as with a number of other parts of implementation, to just illustrate certain general capabilities (one that I know some people have been able setup to use successfully).  Also, as mentioned, the library is unfortunately not longer being maintained it seems....I (or some contributor :) ) probably needs to track down a more up to date fork of this or perhaps re-work based on updated xcframework generation support that has been added as core capability since this.

```
  // all iOS targets (== device and simulator) with minimum version 13
  iOS { v("13") }
  ```
